### PR TITLE
updating app and pyproject

### PR DESCRIPTION
**Pull request by afansi**
update app.yml and pyprojecttoml to reflect module chloe using poetry

### CODE REVIEW

File: app.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-name: casande
+name: chloe
```
Comment (RishabGoel) [inline app.yml:1]: is it absolutely necessary to change the name of the project?

File: pyproject.toml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 [tool.poetry]
-name = "casande"
+name = "chloe"
 version = "0.1.0"
 description = ""
-authors = ["dialogue"]
+authors = ["anonymous"]
```
Comment (RishabGoel) [inline pyproject.toml:5]: Is this done to anonymize the repo.?

**Review by RishabGoel — CHANGES_REQUESTED** [review]

File: pyproject.toml (hunk 2 above)
Comment (afansi) [reply pyproject.toml:5]: not really but it will contribute to

File: app.yml (hunk 1 above)
Comment (afansi) [reply app.yml:1]: yep otherwise, poetry wont be able to get the module chloe in the import

**Review by RishabGoel — APPROVED** [review]

**Pull request merged**